lib/ebaze_web/router.ex

```diff
@@ -42,6 +42,8 @@ defmodule EbazeWeb.Router do
 
     resources "/auctions", AuctionController, except: [:index]
 
+    get "/sell", AuctionController, :sell
```

Comment:
Is this the proper name for this route? Should it be `my-auctions` or `/profile/auctions`?